pkg/guestagent/kubernetesservice/kubernetesservice.go

```diff
@@ -54,16 +54,15 @@ func (s *ServiceWatcher) getServiceInformer() cache.SharedIndexInformer {
 }
 
 func (s *ServiceWatcher) Start() {
-	retryInterval := 10 * time.Second
-	//nolint:staticcheck // SA1019: wait.PollInfinite is deprecated
-	wait.PollInfinite(retryInterval, func() (done bool, err error) {
+	const retryInterval = 10 * time.Second
+	const pollImmidiately = false
```

Comment:
```suggestion
	const pollImmediately = false
```

Reply:
My terrible vision 😂，updated in [874ddf4](https://github.com/lima-vm/lima/pull/1748/commits/874ddf4caec1cbb91ac8581828de2cde58a87bcc)